reference/library/src/webapp/js/headscripts.js

```diff
@@ -817,15 +816,6 @@ function rewriteWebLearnHref(link) {
     }
 }
 
-// Sets the target property to '_blank' so that insecure content is opened in a new window
-function addTargetBlank(link) {
-    if(link.href && link.href.match(/^http:/)) {
```

Comment:
This fix is supposed to remove target=_blank from links created in in the ckeditor in the Overview tool -> Edit.